.github/workflows/update-syscalls.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           path: libs
 
       - name: Bump syscalls
-        uses: falcosecurity/syscalls-bumper@latest
+        uses: falcosecurity/syscalls-bumper@main
```

Comment:
Use main; this is our own repo and it will use assets from latest syscalls-bumper release internally anyway.